tests/query/tests_QuadPattern.cpp

```diff
@@ -190,3 +190,8 @@ TEST_CASE("QuadPattern - Check for iterators and const-iterators") {
         CHECK(*(--e_const_itr) == graph1);
     }
 }
+
+TEST_CASE("null-graph string repr") {
+    Quad const q{Node{}, IRI{"http://test.com#s"}, IRI{"http://test.com#p"}, IRI{"http://test.com#o"}};
```

Comment:
Quad::valid would return false here, right?

Reply:
yes

Reply:
I think we need to fix this API in general, it does not make sense that we can even construct invalid quads, but thats something for another PR I think